packages/@lightningjs/ui-components/src/components/TabBar/TabBar.stories.js

```diff
@@ -267,3 +268,90 @@ CustomTabs.args = {
   alphaSelectedTab: false
 };
 CustomTabs.argTypes = {};
+
+class ColumnContainer extends Base {
+  static get __componentName() {
+    return 'ColumnContainer';
+  }
+  static get tags() {
+    return [
+      'ClippingOffsetContainer',
+      { name: 'Column', path: 'ClippingOffsetContainer.Column' }
+    ];
+  }
+  static _template() {
+    return {
+      ClippingOffsetContainer: {
+        w: 1920 - 160,
+        h: 500,
+        clipping: true,
+        y: context.theme.layout.gutterY.sm * -1,
+        x: context.theme.layout.marginX * -1,
+        Column: {
+          type: Column,
+          y: context.theme.layout.gutterY.sm,
+          x: context.theme.layout.marginX,
+          items: [
+            {
+              type: Row,
+              items: tilesA,
+              autoResizeHeight: true
+            },
+            {
+              type: Row,
+              items: tilesA,
+              autoResizeHeight: true
+            },
+            {
+              type: Row,
+              items: tilesA,
+              autoResizeHeight: true
+            },
+            {
+              type: Row,
+              items: tilesA,
+              autoResizeHeight: true
+            }
+          ]
+        }
+      }
+    };
+  }
+  _getFocused() {
+    return this._Column;
+  }
+}
+
+export const MultipleRows = () =>
+  class MultipleRows extends lng.Component {
+    static _template() {
+      return {
+        TabBar: {
+          type: TabBarComponent,
+          w: 1920 - 160,
```

Comment:
you should actually be able to reference `context.theme.layout.marginX * 2` here. Or even `getColumnWidth(1)`

Reply:
couldn't find `getColumnWidth` in the utils, so used `context.theme.layout.marginX * 2`